test/unit/archipelago-adapter.spec.ts

```diff
@@ -49,9 +49,9 @@ describe('archipelago-adapter', () => {
                 authResponse.resolve(message.authResponse)
                 break
             }
-          }
+          },
```

Comment:
I think this is going against the prettier config... there are many changes like this one.

Reply:
weird, I was trying the lint:fix script, will take a look